.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -25,10 +25,17 @@ body:
     attributes:
       label: Operating system
       description: "Which operating system are you on? Please provide the version as well. If you are on a Mac, please specify Apple silicon or Intel."
-      placeholder: "macOS Monterey 12.2.1 (Intel)"
+      placeholder: "macOS Ventura 13.3 (Arm), Windows 11"
     validations:
       required: true 
 
+  - type: input
```

Comment:
Can't it be a dropdown ?

Reply:
Yes we can have an extra option Other

Reply:
yes but how do you fill the field for 'other' ? it means to add an extra field with user input ?

Reply:
I would say let's start without an extra field and watch if Other is widely used or not

Reply:
ok fixed 👍